core/platform/auth/firebase_auth_services.py

```diff
@@ -374,6 +379,11 @@ def _suppress_firebase_errors():
 
     Errors raised by the SDK are logged and then discarded.
 
+    We suppress exceptions because we do not want the errors to disrupt the
```

Comment:
You could just handle this at the controller level, right?

Presumably we should have the full exception trace in our logs, in case the issue is something with Firebase itself that we need to investigate. 'Firebase Admin SDK raised an exception!' just hides everything...

Reply:
As discussed, this does print the full stack trace.

Marking this as resolved since the other thread holds the main concern (re: should we do this at all?)